docs/user/convert/parsing_and_validation.rst

```diff
@@ -191,17 +191,26 @@ evaluation context, or can be exposed through reporters configured in the conver
       }
     }
 
-Reporters are available for `SLF4J <https://www.slf4j.org/>`__, `Graphite <https://graphiteapp.org/>`__ and
-`Ganglia <http://ganglia.sourceforge.net/>`__. To use Graphite or Ganglia, you must include a dependency on
-``geomesa-convert-metrics-graphite_2.11`` or ``geomesa-convert-metrics-ganglia_2.11``, respectively. Note that
-using Ganglia requires an additional GPL-licensed dependency ``info.ganglia.gmetric4j:gmetric4j``, which is excluded
-by default. The reporters can be configured as follows:
+Reporters are available for `SLF4J <https://www.slf4j.org/>`__, `CloudWatch <https://aws.amazon.com/cloudwatch/>`__,
+`Graphite <https://graphiteapp.org/>`__, and `Ganglia <http://ganglia.sourceforge.net/>`__. To use CloudWatch,
+Graphite, or Ganglia, you must include a dependency on ``geomesa-convert-metrics-cloudwatch_2.11``, 
+``geomesa-convert-metrics-graphite_2.11``, or ``geomesa-convert-metrics-ganglia_2.11``, respectively. 
+Note that using Ganglia requires an additional GPL-licensed dependency ``info.ganglia.gmetric4j:gmetric4j``, 
+which is excluded by default. The reporters can be configured as follows:
 
 ::
 
     geomesa.converters.myconverter {
       options {
         reporters = {
+          cloudwatch = {
+            type           = "cloudwatch"
+            units          = "MILLISECONDS"
+            interval       = "60 seconds"
+            namespace      = "geomesa"
+            withZeroValues = true
+            withRawCounts  = false
```

Comment:
could we change these two params to be dash-separated, to be consistent with the other params? e.g `with-raw-counts`, or even just `raw-counts`.